packages/create-llama/templates/components/vectordbs/typescript/none/index.ts

```diff
@@ -32,13 +34,22 @@ async function getDataSource(llm: LLM) {
   });
 }
 
-export async function createChatEngine(llm: LLM) {
+export async function createChatEngine(llm: OpenAI) {
```

Comment:
@thucpn if the user doesn't select a tool in (questions.ts) then we will still keep the `ContextChatEngine`, so we need two components: one using ContextChatEngine and one the OpenAIAgent

Reply:
that's why Python has two chat engines, see https://github.com/run-llama/LlamaIndexTS/tree/main/packages/create-llama/templates/components/engines/python